app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -362,8 +362,28 @@ def parse_container(container, pod_id)
         :backing_ref     => container.containerID,
         :container_image => parse_container_image(container.image, container.imageID)
       }
+      state_attributes = parse_container_state container.lastState
+      state_attributes.each { |key, val| h[key.to_s.prepend('last_').to_sym] = val } if state_attributes
+      h.merge! parse_container_state container.state
+    end
+
+    def parse_container_state(state_hash)
```

Comment:
Incidentally, I did just that. I'm trying to remember why I abandoned it and changed it to be this way and I'm pretty sure I couldn't get parse_container_state simple enough  for rubocop's taste that way ( 'assignment branch condition ...'). If you want me to give another go let me know, maybe you could help with rubocop this time...
10x